### PR TITLE
algin tcp_reporting_duration default value

### PR DESCRIPTION
**Pull request by zirain**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: source/extensions/filters/http/istio_stats/config.proto [hunk 1]
```diff
@@ -100,7 +100,7 @@ message PluginConfig {
   bool disable_host_header_fallback = 6;
 
   // Optional. Allows configuration of the time between calls out to for TCP
-  // metrics reporting. The default duration is `15s`.
+  // metrics reporting. The default duration is `5s`.
```
Comment (lei-tang) [inline source/extensions/filters/http/istio_stats/config.proto:103]: Where does the 5s value come from? Why does it need to be changed from 15s to 5s? Any performance impact for changing from 15s to 5s?

**Review by kyessenov — APPROVED** [review]

File: source/extensions/filters/http/istio_stats/config.proto (hunk 1 above)
Comment (lei-tang) [reply source/extensions/filters/http/istio_stats/config.proto:103]: Thanks!

**Review by lei-tang — APPROVED** [review]

**Pull request merged**